src/area.rs

```diff
@@ -71,9 +81,19 @@ impl<F: Copy, P, B: MappingBackend<F, P>> MemoryArea<F, P, B> {
     pub const fn backend(&self) -> &B {
         &self.backend
     }
+
+    /// Changes the flags.
+    pub fn set_flags(&mut self, new_flags: F) {
```

Comment:
Change to `pub(crate) const fn`, and move it into the bellow private implementation block.